ssh/tunnel.py

```diff
@@ -81,6 +82,9 @@ def tunnel(user: str, key_path: str, host: str, port: int=22):
     logger.debug('Closing SSH Tunnel: ' + ' '.join(close_tunnel))
     check_call(close_tunnel)
 
+    if os.path.exists(tunnel_socket.name):
+        os.remove(tunnel_socket.name)
```

Comment:
Why is this wrapped in a conditional, is there a possibility that `tunnel_socket` will not exist?